<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>redo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>flip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paintBucketButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>paint bucket</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="gridCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>grid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previewButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>preview</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFrameBut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="frameScrollBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="shapeButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>draw shape</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameCounter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Current Frame: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="newProjectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Make New Project</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeOptions">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string comment="8">8 Bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="16">16 Bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="32">32 Bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="64">64 Bit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="projectSizeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Project Size:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorDialogButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Picker</string>
    </property>
   </widget>
   <widget class="QComboBox" name="shapeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rectangle</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="previousColorButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Color Selector</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Eraser">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Eraser</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
